app/controllers/patients_controller.rb

```diff
@@ -507,18 +515,6 @@ def household_removeable
     render json: { removeable: !duplicate_contact }
   end
 
-  def send_reminder
```

Comment:
I actually removed this in one of my PRs - should be fine though git wise (but if we do see conflicts, this would be a likely culprit).